src/components/lightbox/LightBoxPlugin.tsx

```diff
@@ -282,9 +283,19 @@ const MyComponent = () => {
 						metaData?.img_caption ||
 						metaData?.ocr) && (
 						<motion.div
-							animate={{ y: isExpanded ? 0 : "calc(100% - 100px)" }}
+							animate={{
+								y: isExpanded
+									? 0
+									: `calc(100% - ${
+											currentBookmark?.addedTags?.length > 0 ? 145 : 110
+										}px)`,
+							}}
```

Comment:
Can you extract this, instead of duplicating?

```
const collapsedOffset = currentBookmark?.addedTags?.length > 0 ? 145 : 110;
const collapsedY = `calc(100% - ${collapsedOffset}px)`;

<motion.div
  animate={{ y: isExpanded ? 0 : collapsedY }}
  initial={{ y: collapsedY }}
  // ...
>
```